File: 0412-Fizz_Buzz.py
class Solution:
  def fizzBuzz(self, n: int) -> list[str]:
    answer = []
    for i in range(1, n+1):
      if n % 15 == 0:
        answer.append("FizzBuzz")
      elif n % 5 == 0:
        answer.append("Buzz")
      elif n % 3 == 0:
        answer.append("Fizz")
      else:
        answer.append(str(n))
    return answer

File: 0412-fizz_buzz.go
func fizzBuzz(n int) []string {
	answer := make([]string, n)
	for i := 1; i <= n; i++ {
		if i%15 == 0 {
			answer[i-1] = "FizzBuzz"
		} else if i%3 == 0 {
			answer[i-1] = "Fizz"
		} else if i%5 == 0 {
			answer[i-1] = "Buzz"
		} else {
			answer[i-1] = strconv.Itoa(i)
		}
	}
	return answer
}